src/pages/index.js

```diff
@@ -6,171 +6,48 @@ import ListItem from "../components/listItem"
-        chronology="1"
-        category="deployment"
-        level="2"
-        comment="Release Cycle, alpha, beta, polished, external, prod..."
-      />
-      <ListItem
-        name="Write unit tests"
-        chronology="2"
-        category="reliability"
-        level="3"
-      />
-      <ListItem
-        name="Monitoring battery / network usage"
-        chronology="2"
-        category="monitoring"
-        level="2"
-      />
-      <ListItem
-        name="Crash / error tracing"
-        chronology="3"
-        category="troubleshooting"
-        level="3"
-      />
-      <ListItem
-        name="Be forward compatible"
-        chronology="2"
-        category="availability"
-        level="3"
-        comment="Considering the scale and lack of control in the mobile environment, managing changes in a safe and reliable manner is arguably one of the most critical pieces of managing a reliable mobile application."
-      />
-      <ListItem
-        name="Be GDPR compliant"
-        chronology="3"
-        category="security"
-        level="3"
-        comment="If your app is available in Europe."
-      />
-      <ListItem
-        name="Staged rollout / Phased Releases"
-        chronology="3"
-        category="reliability"
-        level="3"
-        comment="Define a progressive release life cycle"
-      />
-      <ListItem
-        name="Dogfooding / Internal testing"
-        chronology="3"
-        category="reliability"
-        level="3"
-      />
-      <ListItem
-        name="Test all rollback toggles / killswitch before deploying"
-        chronology="3"
-        category="reliability"
-        level="3"
-      />
-      <ListItem
-        name="Add caching"
-        chronology="2"
-        category="availability"
-        level="2"
-      />
-      <ListItem
-        name="Add documentation"
-        chronology="2"
-        category="documenting"
-        level="3"
-      />
-      <ListItem
-        name="Post-mortems"
-        chronology="4"
-        category="monitoring"
-        level="1"
-      />
-      <ListItem
-        name="Implement CI/CD"
-        chronology="2"
-        category="deployment"
-        level="3"
-      />
-      <ListItem
-        name="Runbook, configure alerting"
-        chronology="3"
-        category="recoverability"
-        level="2"
-      />
-      <ListItem
-        name="Support prod / test mode switch"
-        chronology="2"
-        category="reliability"
-        level="2"
-      />
-      <ListItem
-        name="Be immutable"
-        chronology="2"
-        category="reliability"
-        level="3"
-      />
-      <ListItem
-        name="Support i8n"
-        chronology="2"
-        category="reliability"
-        level="2"
-      />
-      <ListItem
-        name="Consider risk / criticity"
-        chronology="3"
-        category="security"
-        level="3"
-      />
-    </List>
-  </Container>
-)
+class App extends React.Component {
+
+  state = {
```

Comment:
I would not put that in the React state property but create a new one, this not need to be stated